site/content/en/docs/Concepts/services.md

```diff
@@ -100,7 +100,7 @@ The secrets section let's you pass in secret values to your service as environme
 
 #### Environments
 
-The environment section lets you overwrite any value in your manifest based on the environment you're in. In the example manifest above, we're overriding the _count_ parameter so that we can run 2 coppies of our service in or _prod_ environment.
+The environment section lets you overwrite any value in your manifest based on the environment you're in. In the example manifest above, we're overriding the _count_ parameter so that we can run 2 coppies of our service in our _prod_ environment.
```

Comment:
```suggestion
The environment section lets you overwrite any value in your manifest based on the environment you're in. In the example manifest above, we're overriding the _count_ parameter so that we can run 2 copies of our service in our _prod_ environment.
```